### DISCUSSION TITLE
Nextcloud not working with http

### DISCUSSION THREAD

**Discussion by Jannik44** [nextcloud/all-in-one#3986] [Questions] [answered]
### Steps to reproduce
1. install it
2. use http

### Expected behavior 
it should let me use http

### Actual behavior 
it forces me to use https

**Comment by Zoey2936** [comment]
I think I shouldn't explain why using http is a very bad Idea

**Comment by Jannik44** [comment]
it isnt, at least locally

**Comment by Zoey2936** [comment, marked as answer]
see https://github.com/nextcloud/all-in-one/blob/main/local-instance.md

**Comment by joshtrichards** [comment]
https://github.com/nextcloud/all-in-one/discussions/639
